docs/source/using/addons.adoc

```diff
@@ -31,65 +31,60 @@ oc adm policy add-scc-to-group anyuid system:authenticated
 ----
 <1> (Required) Name of the add-on.
 <2> (Required) Description of the add-on.
-<3> Actual add-on command. In this case, the command executes the _oc_ binary. For the full
-list of the supported add-on commands, see the <<addon-commands>> section.
+<3> Actual add-on command. In this case, the command executes the *oc* binary.
 
 NOTE: Comment lines can be inserted at anywhere in the file. Comment lines must start with the '#' character.
 
-Enabled add-ons (see <<enabling-disabling-addons>>) are be applied during
-link:../command-ref/minishift_start{outfilesuffix}[`minishift start`], immediately after the initial
-cluster provisioning is successfully completed.
+Enabled add-ons are applied during xref:../command-ref/minishift_start.adoc#minishift-start[`minishift start`], immediately after the initial cluster provisioning is successfully completed.
 
 [[addon-commands]]
-== Add-on commands
+== Add-on Commands
 
 This section describes the command types that an add-on file can contain and what the commands do.
 The following command types are available forming a sort of mini-DSL for add-ons:
 
 ssh::
-If the add-on command starts with _ssh_, you can run any command within the Minishift-managed VM.
-This is similar to running link:../command-ref/minishift_ssh{outfilesuffix}[`minishift ssh`] and
-then executing any command on the VM.
+If the add-on command starts with `ssh`, you can run any command within the Minishift-managed VM.
+This is similar to running xref:../command-ref/minishift_ssh.adoc#minishift-ssh[`minishift ssh`] and then executing any command on the VM.
+For more information about the `minishift ssh` command usage, see xref:../using/managing-minishift.adoc#connecting-with-ssh[Connecting to the Minishift VM with SSH].
 
 oc::
-If the add-on command starts with _oc_, it uses the _oc_ binary that is cached on your host to execute
-the specified `oc` command. This is similar to running `oc --as system:admin ...` from the command-line.
+If the add-on command starts with `oc`, it uses the *oc* binary that is cached on your host to execute the specified `oc` command.
```

Comment:
I think it formatting should be the other way around. First use of oc \*oc\* (here we are referring to the "text" oc), second use \`oc\` since we are referring to the binary/command